designer/server/src/main/scala/pl/touk/nussknacker/ui/definition/UIProcessObjectsFactory.scala

```diff
@@ -224,7 +218,6 @@ object UIProcessObjectsFactory {
         }
         .toMap
 
-    val transformed = modelDefinition.transform(createUIComponentDef)
```

Comment:
This was unused. The `transform` function was used only in one test so I've removed it, unless it's needed for convenience?